tests/test_sim2.py

```diff
@@ -195,6 +225,34 @@ def test_cannot_set_zero_scale() -> None:
         Sim2(R, t, s)
 
 
+def test_sim2_theta_deg_1() -> None:
+    """Ensure we can recover the rotation angle theta"""
```

Comment:
Nit: period and end of sentence for consistency.

Reply:
thanks, should be updated now